### PR TITLE
fix broke link in api

### PR DESCRIPTION
**Pull request by u2takey**
**What this PR does / why we need it**:
fix some broken link
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/batch/v1/types.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ type JobSpec struct {
 	// and other jobs to not function correctly.  However, You may see
 	// `manualSelector=true` in jobs that were created with the old `extensions/v1beta1`
 	// API.
-	// More info: https://git.k8s.io/community/contributors/design-proposals/selector-generation.md
+	// More info: https://git.k8s.io/community/blob/master/contributors/design-proposals/apps/selector-generation.md
```
Comment (xiangpengzhao) [inline staging/src/k8s.io/api/batch/v1/types.go]: Did you try to visit this link? It doesn't work. The `blob/master/` is redundant here.

Comment (xiangpengzhao) [reply staging/src/k8s.io/api/batch/v1/types.go]: Please modify other links below as well :)

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

**Comment by xiangpengzhao** [comment]
/lgtm

Thanks!

**Comment by nikhita** [comment]
These are legit errors. Example:

```
I0920 12:04:55.393] -      "description": "Items is a list of LimitRange objects. More info: https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md",
I0920 12:04:55.394] +      "description": "Items is a list of LimitRange objects. More info: https://git.k8s.io/community/contributors/design-proposals/admission_control_limit_range.md",
```

The comments are also used in the generated files. You'll need to use the `hack/update-*` scripts to update the comments in the generated files as well.

**Comment by CaoShuFeng** [comment]
I think https://github.com/kubernetes/community/pull/1010 breaks a lot others.
Can you fix them all?

Some links show in docstring, so we need to run hack/update-all.sh to regenerate some codes.

**Review by janetkuo — COMMENTED** [review]
@u2takey do you plan to continue on this PR? Auto-generated files need to be updated to make verification pass. The easiest way to do this is running `hack/update-all.sh`

Please update `staging/src/k8s.io/api/extensions/v1beta1/types.go` too (links to https://git.k8s.io/community/contributors/design-proposals/security_context.md)

**Comment by u2takey** [comment]
Sorry for late update. Updated with hack/update-all.sh @janetkuo

**Comment by nikhita** [comment]
I tried testing this locally - `verify-swagger-spec` passes. (maybe related to https://github.com/kubernetes/kubernetes/issues/53481?)

/retest

**Comment by CaoShuFeng** [comment]
Please run:
hack/update-swagger-spec.sh

**Comment by u2takey** [comment]
@CaoShuFeng not work, i need rebase first?

**Comment by CaoShuFeng** [comment]
>  i need rebase first?

It's right direction, I guess.

**Comment by u2takey** [comment]
> `It's right direction, I guess.`
not work still

**Comment by u2takey** [comment]
> It's right direction, I guess.

still not work

**Comment by CaoShuFeng** [comment]
But it works in my environment.
http://paste.openstack.org/show/623191/

**Comment by CaoShuFeng** [comment]
I have a bad suggestion:
you can modify api/swagger-spec/apps_v1.json manually.

**Comment by nikhita** [comment]
@u2takey I created a PR on your branch (https://github.com/u2takey/kubernetes/pull/1) to update the swagger spec. Can you please merge that and squash the commits into a single commit? Thanks!

**Comment by u2takey** [comment]
ping @nikhita for approve

**Comment by nikhita** [comment]
@u2takey I cannot give approval. :)

/assign @smarterclayton for approval

**Comment by nikhita** [comment]
@u2takey Sorry that fixing links is turning out to be a tedious experience. :(

**Comment by u2takey** [comment]
@nikhita that's ok. I found some broken links already fixed in master, closed this one.

**Pull request closed without merging**